glow.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, n_in_channels, n_mel_channels, n_layers, n_channels,
         self.n_layers = n_layers
         self.n_channels = n_channels
         self.in_layers = torch.nn.ModuleList()
-        self.res_layers = torch.nn.ModuleList()
+        self.res_skip_layers = torch.nn.ModuleList()
         self.skip_layers = torch.nn.ModuleList()
```

Comment:
this layer (self.skip_layers) is not being used anymore. Let's remove it from __init__, forward and anywhere where it's not necessary.